src/controllers/queries/index.ts

```diff
@@ -246,32 +312,22 @@ export class QueriesController extends HTMLController {
       companyId: UUID
       action: 'success'
       totalScope3CarbonEmissions: string
-      partialResponse?: number
     }
   ): Promise<HTML> {
     this.logger.debug('query page requested')
 
-    const [connection]: ConnectionRow[] = await this.db.get(
-      'connection',
-      { id: body.companyId, status: 'verified_both' },
-      [['updated_at', 'desc']]
-    )
+    const [connection] = await this.db.get('connection', { id: body.companyId, status: 'verified_both' }, [
+      ['updated_at', 'desc'],
+    ])
     if (!connection) {
-      throw new InvalidInputError(`Invalid connection ${body.companyId}`)
+      throw new NotFoundError(`Invalid connection ${body.companyId}`)
     }
     if (!connection.agent_connection_id || connection.status !== 'verified_both') {
-      throw new InvalidInputError(`Cannot query unverified connection`)
```

Comment:
should this not remain as `InvalidInputError` since it is found? like in here: https://github.com/digicatapult/veritable-ui/pull/137/files#r1736140218 ?

Reply:
well, simply put we would use notfound when the requested resource is not available e.g. `/something/{resource1}/anotherthing/{resource2} - so in this instance we would throw not found but if we are claiming a resource from `resource1` properties, not sure if it makes sense